builder/dockerfile/builder.go

```diff
@@ -208,13 +209,30 @@ func newBuilder(clientCtx context.Context, options builderOptions) *Builder {
 	return b
 }
 
+// Build 'LABEL' command(s) from '--label' options and add to the last stage
+func buildLabelOptions(labels map[string]string, stages []instructions.Stage) {
+	if len(labels) > 0 {
+		keys := []string{}
+		for key := range labels {
+			keys = append(keys, key)
+		}
+
+		// Sort the label to have a repeatable order
+		sort.Strings(keys)
+		for _, key := range keys {
+			value := labels[key]
+			// Value must be single quoted to prevent env variable expansion
```

Comment:
Instead of this quoting hack, could we add `NewLabelCommand(key, value string, expand bool)` so that when `expand` is false it would make `label.Expand()` no-op.

Reply:
Thanks very much for the review! 👍  Will update the change per the comment...